### ISSUE TITLE
Extension not working (only syntax highlighting that is working)

### ISSUE THREAD

**Issue by raveltan** [sveltejs/language-tools#618]
**Describe the bug**
Extension is unable to provide a proper autocomplete, it only provides a syntax highlighting,
all svelte commands like restart lang server is also failing even after custom path to lang server is set

**To Reproduce**
Install extension on vscode,
create a new project
navigate to App.svelte
try the autocomplete or the command such as restart lang server

**Expected behavior**
It should show proper autocomplete and formatting capabilites.

**System (please complete the following information):**
 - OS: Pop Os 20.04
 - IDE: VScode
 - Plugin/Package: Svelte for VSCode

**Comment by jasonlyu123** [comment]
Can you check the output channel at the bottom and choose svelte in the bottom right dropdown at see if there are any errors logged?

**Comment by raveltan** [comment]
```bash
[2020-10-18 11:10:24.407] [exthost] [error] Activating extension svelte.svelte-vscode failed due to an error:
[2020-10-18 11:10:24.419] [exthost] [error] Error: Cannot find module 'semver'
Require stack:
- /home/ravel/.vscode/extensions/svelte.svelte-vscode-102.2.0/node_modules/vscode-languageclient/lib/main.js
- /home/ravel/.vscode/extensions/svelte.svelte-vscode-102.2.0/dist/src/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:844:17)
	at Module._load (internal/modules/cjs/loader.js:737:27)
	at Function.Module._load (electron/js2c/asar.js:769:28)
	at Function.t._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1057:776)
	at Function.i._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1027:486)
	at Function.n._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1023:767)
	at Module.require (internal/modules/cjs/loader.js:899:19)
	at r (/usr/share/code/resources/app/out/vs/loader.js:17:346)
	at Object.<anonymous> (/home/ravel/.vscode/extensions/svelte.svelte-vscode-102.2.0/node_modules/vscode-languageclient/lib/main.js:12:16)
	at Module.i._compile (/usr/share/code/resources/app/out/vs/loader.js:17:571)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1051:10)
	at Module.load (internal/modules/cjs/loader.js:862:32)
	at Module._load (internal/modules/cjs/loader.js:774:14)
	at Function.Module._load (electron/js2c/asar.js:769:28)
	at Function.t._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1057:776)
	at Function.i._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1027:486)
	at Function.n._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1023:767)
	at Module.require (internal/modules/cjs/loader.js:899:19)
	at r (/usr/share/code/resources/app/out/vs/loader.js:17:346)
	at Object.<anonymous> (/home/ravel/.vscode/extensions/svelte.svelte-vscode-102.2.0/dist/src/extension.js:27:33)
	at Module.i._compile (/usr/share/code/resources/app/out/vs/loader.js:17:571)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1051:10)
	at Module.load (internal/modules/cjs/loader.js:862:32)
	at Module._load (internal/modules/cjs/loader.js:774:14)
	at Function.Module._load (electron/js2c/asar.js:769:28)
	at Function.t._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1057:776)
	at Function.i._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1027:486)
	at Function.n._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1023:767)
	at Module.require (internal/modules/cjs/loader.js:899:19)
	at require (internal/modules/cjs/helpers.js:74:18)
	at Function.s [as __$__nodeRequire] (/usr/share/code/resources/app/out/vs/loader.js:34:963)
	at p._loadCommonJSModule (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:1028:771)
	at p._doActivateExtension (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:925:742)
	at p._activateExtension (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:924:822)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async Promise.all (index 0)
```

**Comment by jasonlyu123** [comment]
Seems like some dependence is not installed. Don't know if that's a deploy issue or installation issue. Maybe you can try reinstall the extension.

**Comment by raveltan** [comment]
Reinstalled and still no success

**Comment by dummdidumm** [comment]
Very strange. I did a fresh install of the extension and cannot reproduce this. I looked into the VSIX of the extension and `semver` is in there. Could you look up the path where your extension is installed, look into the node_modules folder (essentially the one that's in your error message stack) and see if the `semver` is in there? Did it work with a previous extension version?

**Comment by raveltan** [comment]
yup, for some reason semver is not there, here is the contents of the node_modules

```bash
chokidar     estree-walker  svelte-language-server
cosmiconfig  lodash         vscode-languageclient
```

Before this happen, the extension actually had worked properly, but i'm not sure if it's a previous version of the extension or the same version.

**Comment by dummdidumm** [comment]
Very strange, that's definitely too few modules. If you revert to an older version, does the error persist? You can also try downloading the extension from the website and unpackaging it into the folder.

**Comment by raveltan** [comment]
I've tried several different version and the result is still the same, i even try to reinstall vscode and still the problem presist.
i assume that the problem is specific to my machine,
so i'm switching to webstorm (there is no problem with svelte extension there)

Thanks for the help

**Comment by Tekameikite** [comment]
I have the same problem, on the same system. Installing it from the marketplace through unpackaging it doesn't help. It's really weird because it stopped working when I switched to Pop OS from Manjaro - on which it was working perfectly fine.

**Comment by 0xNF** [comment]
The Svelte extension suffers from the same problem on my Windows machine - no Svelte intellisense, and restarting the Svelte server just gives me "Command 'Svelte: Restart Language Server' resulted in an error (command 'svelte.restartLanguageServer' not found)".

**Comment by dummdidumm** [comment]
So ... why did you set the `ls-path` then? The concatenation should only occur if you set the `ls-path` setting. What did you set it to?

**Comment by 0xNF** [comment]
You're right, I did have modified values for my ls-path. I must have forgotten to undo them - I had encountered this same error, svelte not offering intellisense, a few months back and was fiddling around with those settings in hopes it would fix it. At the time, it didn't change anything, and I was left with a broken installation. 

After you pointed that out, I went back and set the ls-path back to empty, restored the `ls_path ||` to the server module line, restarted vs code and it worked fine.

While looking for a solution to this problem yesterday, I did notice on some other thread that I can't find at the moment that there was a requirement for a certain, fairly recent version of VS Code, I think for 1.48.x or higher. It's possible that when I first found the issue I was on a version of code that wasn't supported, but now I am.
